app/src/main/kotlin/co/netguru/baby/monitor/client/feature/machinelearning/MachineLearning.kt

```diff
@@ -43,7 +43,7 @@ class MachineLearning(context: Context) {
     }
 
     companion object {
-        internal const val DATA_SIZE = 176_400
+        internal const val DATA_SIZE = 4 * SAMPLING_RATE
```

Comment:
Why 4? Consider moving this value to constants

Reply:
I guess it should be self explanatory as SAMPLING_RATE is number of samples per second. So DATA_SIZE is a recording that has 4 seconds.

Reply:
Well, not for me at first sight 😅